web/src/main/java/org/openmrs/web/controller/patient/ShortPatientFormValidator.java

```diff
@@ -129,7 +129,7 @@ public void validate(Object obj, Errors errors) {
 			if (OpenmrsUtil.nullSafeEquals(possibleDuplicate.getId(), personAddress.getId()))
 				continue;
 			
-			if (!possibleDuplicate.isBlank() && !personAddress.isBlank()
+			if (!possibleDuplicate.isVoided() && !possibleDuplicate.isBlank() && !personAddress.isBlank()
```

Comment:
Hi @dkayiwa , thanks for the reminder. :-) Here is the unit test. Thank you for reviewing this pull request.